public/help.js

```diff
@@ -26,7 +28,7 @@ if (expand)
 			})
 	})
 
-if (window.location.pathname === '/help/visitor-counter') {
+if (window.location.pathname == basePath + '/help/visitor-counter') {
```

Comment:
Is there a reason to change the `===` to `==` here, or is it an accident?

Reply:
Accident, fixed.